db/table_cache.cc

```diff
@@ -336,10 +337,21 @@ Status TableCache::Get(const ReadOptions& options,
 
     if (auto row_handle =
             ioptions_.row_cache->Lookup(row_cache_key.GetUserKey())) {
+
+      // Cleanable routine to release the cache entry
+      Cleanable cache_entry_clean;
```

Comment:
If you click on the link that is pasted in my comment: 
> Can you make sure to consider this comment? #2492 (comment)
it will take you to this comment:
> To be consistent with the existing code, lets call it value_pinner. Lets rename it on other places too.